lib/datastore/request.js

```diff
@@ -196,6 +196,76 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  *   data: {
  *     rating: '10'
  *   }
+ * }, function(err) {
+ *   console.log(key.path); // [ 'Company', 5669468231434240 ]
+ *   console.log(key.namespace); // undefined
+ * });
+ *
+ * //-
+ * // Save a single entity using a provided name instead of auto-generated ID.
+ * //
+ * // Here we are providing a key with name instead of an id. After saving, the
```

Comment:
Total nit, but should we use `ID` or `id`? Just like when I write docs, they are both used interchangeably here.